packages/react-union/src/scanning/descriptors/cheerio.js

```diff
@@ -0,0 +1,36 @@
+import o from 'ramda/src/o';
+import unary from 'ramda/src/unary';
+import { mergeDeepRightAll } from '../../utils';
+
+const dangerouslyParseJsonContent = o(unary(JSON.parse), wrapper => wrapper.html());
+
+const parseJsonContent = wrapper => {
```

Comment:
I know that this file and the dom.js file are very similar in their layout, but I don't think it's worth it to refactor them into a single file, as they have very many subtle differences. Think of them as sharing a common interface.